docs/tools/new_tools_temp.md

```diff
@@ -0,0 +1,53 @@
+# New tools - draft
+
+<!--well this is a disaster-->
+
+Starting with version 6.x, Mbed OS is moving from Mbed CLI to a new build tool: it's cmake-based,
```

Comment:
At the moment I'm finding the way this is worded slightly confusing, we say our new "build tool" is cmake-based, then below we say it's Python based. We should make the distinction between CMake and the command line tools clearer.

Reply:
The whole intro is a bit of a mess (hence the "well this is a disaster"). When I get more info on the tool, it'll be easier to know what to say here. But if you can explain the distinction, that'll help.

Reply:
Sorry, I should've added an explanation. At a high level the distinction is:
* Ninja is our build system of choice.
* CMake is an open-source system that generates the build environment and manages the build process in a compiler independent manner.
* mbed-tools (or whatever we end up calling it) is our new command line tool that parses the build configuration from the Mbed configuration system and outputs it in a format CMake can read. mbed-tools will also provide a user friendly interface to CMake and Ninja, allowing the user to configure, generate and execute the build system with a single command.

Hopefully that helps!

Reply:
That will be lovely for the intro page (the next file in this review), and should help me figure out what people are installing here. Thank you!